API-8684: Add info about accesible threads (#928)
livechat/livechat-public-docs: content/messaging/agent-chat-api/v3.4/index.mdx

| `active`                        | required  | Possible values: `true` (thread is still active) or `false`(thread no longer active)                               |
| `events`                        | optional  | Doesn't exists if `restricted_access` is present.                                                                  |
| `properties`                    | optional  |                                                                                                                    |
| `restricted_access`             | optional  | Contains the reason for access restriction.                                                                        |
| `queue`                         | optional  | Present only if the chat is in the queue. The wait time for an available agent is approximated in seconds.         |
| `previous_thread_id`            | optional  | Present only if there is a preceding thread.                                                                       |
| `next_thread_id`                | optional  | Present only if there is a following thread.                                                                       |

> | `previous_accessible_thread_id` | optional  | Present only if there is a preceding thread accessible for user. Only present in [`list_archives`](#list-archives) |

Reviewer: ```suggestion
| `previous_accessible_thread_id` | optional  | Present only if there is a preceding thread accessible for the user. Field present only in [`list_archives`](#list-archives) |
```

> | `next_accessible_thread_id`     | optional  | Present only if there is a following thread accessible for user. Only present in [`list_archives`](#list-archives) |

Reviewer: ```suggestion
| `next_accessible_thread_id`     | optional  | Present only if there is a following thread accessible for the requester. Field present only in [`list_archives`](#list-archives) |
```

> | `next_accessible_thread_id`     | optional  | Present only if there is a following thread accessible for the requester. Field present only in [`list_archives`](#list-archives) |

Reviewer: ```suggestion
| `next_accessible_thread_id`     | optional  | Present only if there is a following thread accessible for the requester. Field present only in [`list_archives`](#list-archives). |
```
